### ISSUE TITLE
The Light and Dark Variants Seem to be Exchanged

### ISSUE THREAD

**Issue by 3N4N** [challenger-deep-theme/xresources#1]
I tried to use your colorscheme from this repo and saw that the dark and light variants of each of the colors, except color0 and color8, are inverted, i.e. the dark variant have the lighter version and vice versa. Is that intended or a mistake?

I manually changed those and now it works fine. I just wanted to let you know.

And, Thanks! for the colorscheme. Appreciate it.

**Comment by MaxSt** [comment]
Thanks for your feedback :+1: . 

> Is that intended or a mistake?

This is intended. The reason is that I prefer the light colors, but in order to bring better contrast to this theme I made the colors 9-15 darker. I inverted color 0 and 8 because a lot of terminal programs use these colors and expect color 0 to be darker than 8. Have you experienced any problems with the light colors in an CLI application?
